packages/lesswrong/components/posts/PostsItem2.tsx

```diff
@@ -174,6 +173,11 @@ export const styles = (theme: ThemeType): JssStyles => ({
       display: "block"
     }
   },
+  nonMobileIcons: {
```

Comment:
This seems the wrong way around? Shouldn't this be `display:none`?

Reply:
Argh, yes. Is immediately obvious on visual inspection of the resulting page. Doing a quick pass through all the pages to make sure there's nothing else similar.